example/follow_me/follow_target.cpp

```diff
@@ -0,0 +1,100 @@
+/**
+* @file follow_target.cpp
+* @brief Example that switches Vehicle mode to FollowMe.
+* @author Author: Shakthi Prashanth M <shakthi.prashanth.m@intel.com>
```

Comment:
In general, I recommend doing integration tests when adding a new features. The reasons are:

- We need integration tests for all the functionality anyway, so we might as well do it while implementing it.
- The example should only cover a simple part of the API to quickly get a developer up to speed.
- It is easier to compile and run it because you don't always need to install the library, and then build the example and link it.
- You can use the internal `LogDebug()` statements.

I propose you move this over to an integration test and add an example at the very end when everything is working.

Oh, I see now you also added an integration test. I would suggest you keep developing there and create a small example at the end.

Reply:
Agree. Done.